src/components/Header.vue

```diff
@@ -5,12 +5,12 @@
   >
     <header>
       <nav class="flex py-6 bg-white items-center justify-between">
-        <div class="flex">
+        <div class="flex items-center">
           <img
             src="@/assets/logo.svg" alt="Logo"
             class="flex-shrink-0 h-12 w-12"
           >
-          <h1 class="text-xl sm:text-xl lg:text-4xl text-gray-900 font-bold ml-3">
+          <h1 class="text-xl text-gray-900 font-bold ml-3">
```

Comment:
Did you test this on mobile after removing? (just to be sure)

Reply:
Yeah, it still looks good on mobile, I think the text being that size on all devices is fine

Reply:
Actually it will look better w/o it there on mobile